src/ocean/text/Util.d

```diff
@@ -785,6 +788,136 @@ bool isSpace(T) (T c)
            return (c <= 32 && (c is ' ' || c is '\t' || c is '\r' || c is '\n' || c is '\f' || c is '\v')) || (c is '\u2028' || c is '\u2029');
 }
 
+/*******************************************************************************
+
+    Enumeration of the different ways in which a string can be padded in order
+    to increase its length.
+
+*******************************************************************************/
+
+public enum PadType : ubyte
+{
+    PadPre,    // Padding is only applied before the string
+    PadPost,   // Padding is only applied after the string
+    PadAround  // Padding is applied on both sides of the string
+}
+
+/*******************************************************************************
+
+    Shortens/Lengthens a string as necessary to get it to a desired length.
```

Comment:
Hmm, from a usage point of view, isn't it more convenient to have a single function which automatically takes care of either padding or truncating as necessary without the programmer having to bother about which one to use?

I could of course have private functions `pad()` & `truncate()`, and call them from the convenience wrapper, but I feel they'll simply be superfluous. The meat of this function really is the lengthening of the original string (because you need to check whether you want to length from the left or the right or on both sides). Also, given the reasonable size of the function as it currently is, splitting it further wouldn't add much value imo.